python_docs_theme/static/pydoctheme_dark.css

```
@@ -1,178 +1,332 @@
-/* Common colours */
+/* Updated Common colors */
```

This change isn't needed
```suggestion
/* Common colours*/
```